Project/Assets/ML-Agents/Examples/Sorter/Scripts/NumberTile.cs

```diff
@@ -8,7 +6,7 @@ public class NumberTile : MonoBehaviour
     public Material DefaultMaterial;
     public Material SuccessMaterial;
 
-    private bool m_Visited = false;
```

Comment:
Not sure how people feel about. My instinct is to a) be explicit (from python) and b) always initialize everything (coming from C++), but Rider likes to complain about it.

Reply:
Yeah I think making them explicit is not a bad thing and sometimes it feels safer (I know it works that way but I like seeing it written that way).
No strong opinions anyway.

Reply:
_traumatized by c++_